src/main/java/com/onlinebanking/config/DevConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.onlinebanking.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import static com.onlinebanking.constant.ProfileTypeConstants.DEV;
+
+/**
+ * This class provides every bean, and other configurations needed
+ * to be used in the development phase.
+ *
+ * @author George on 6/22/2021
+ * @version 1.0
+ * @since 1.0
+ */
+@Configuration
+@Profile(DEV)

```

Comment:
It's always best practice to include the full class to reference the constant instead of static import. 
Eg. (SomeConstant.FIELD) makes it easier to read and know where it's coming from compared to just (FIELD) 

Reference to Josh book concerning it (Effective java)
https://www.iitk.ac.in/esc101/05Aug/tutorial/java/interpack/staticimport.html

Reply:
forgot to choose the revision and approved